lib/wanda/execution/behaviour.ex

```diff
@@ -9,14 +9,14 @@ defmodule Wanda.Execution.Behaviour do
               execution_id :: String.t(),
               group_id :: String.t(),
               targets :: [Target.t()],
-              env :: %{String.t() => String.t()}
+              env :: map()
```

Comment:
We could be more precise in the spec, `%{String.t() => boolean() | number() | String.t()}`